### DISCUSSION TITLE
Free space requirements to distro-sync

### DISCUSSION THREAD

**Discussion by richardjohnjensen** [AlmaLinux/almalinux-deploy#3] [Ideas]
Should a check or note warn users that distro-sync is going to require lots of space to replace OS?
I hadn't consider it with my VirtualBox test... and my little test machine did not convert.

Not sure how to compute required space?

**Comment by ezamriy** [comment]
It's a good question. The amount of free space required depends on a list of installed packages. Theoretically, it is possible to get to know the size of each package but practically it may be kinda slow.

**Reply by richardjohnjensen** [reply]
I have seen other vendors recommending 5GB available in the /var/cache partition. I have incorporated that as a minimal check in my pre-migration wrapper.
